takwimu/models/dashboard.py

```diff
@@ -64,7 +64,6 @@ class TopicPage(Page):
     # Search index configuration
 
     search_fields = Page.search_fields + [
-        index.SearchField('title'),
```

Comment:
Any reason we are dropping `title` from search?

Reply:
@kilemensi title is added by `Page.search_fields`. That was redundant and causing errors